sdks/java/io/amazon-web-services2/src/main/java/org/apache/beam/sdk/io/aws2/common/RetryConfiguration.java

```diff
@@ -68,7 +68,7 @@
   public abstract RetryConfiguration.Builder toBuilder();
 
   public static Builder builder() {
-    return Builder.builder();
+    return Builder.builder().numRetries(3);
```

Comment:
Why number of retries "3" was choosen?

Reply:
That's the default of fallback retry mode ('LEGACY') of the AWS SDK if nothing is configured